core/modules/xrdsvc/StreamBuffer.cc

```diff
@@ -38,17 +38,42 @@ LOG_LOGGER _log = LOG_GET("lsst.qserv.xrdsvc.StreamBuffer");
 }
 
 
+using namespace std;
+
+
 namespace lsst {
 namespace qserv {
 namespace xrdsvc {
 
-std::atomic<size_t> StreamBuffer::_totalBytes(0);
+
+atomic<int64_t> StreamBuffer::_maxTotalBytes{40'000'000'000};
+atomic<int64_t> StreamBuffer::_totalBytes(0);
+mutex StreamBuffer::_createMtx;
+condition_variable StreamBuffer::_createCv;
+
+
+static void StreamBuffer::setMaxTotalBytes(int64_t maxBytes) {
+    LOGS(_log, LOG_LVL_INFO, "StreamBuffer::setMaxTotalBytes maxBytes=" << maxBytes);
```

Comment:
I suggest using (here and elsewhere) the predefined constant `__func__` instead:
```
"StreamBuffer::" + string(__func__) + " setMaxTotalBytes maxBytes=" << maxBytes);
```
This would simplify the code refactoring effort should you (or someone else) decide to rename the metod.

Reply:
I have some mixed feeling about this. If there's just one or two log statements, I don't worry about it. I like to cut and paste comments from the log file and then use that to search the code. If I search for " setMaxTotalBytes maxBytes=" in the code, I wont get any hits after making the above change. Once there gets to be more log statements in the function, it's more worthwhile. In this case, I made the change.